src/reference/modules/reference/pages/variables.adoc

```diff
@@ -0,0 +1,150 @@
+= Variables
+
+== General
+
+In Rudder, you can define and use variables via the technique editor or the differents pre-built techniques.
+
+All variables are defined under a `prefix` (scope), so to reference a variable you will always need its `prefix` and its `name`, separated via a `.` char.
+To call a variable in Rudder we use `${...}` brackets syntax as described below:
+
+----
+// Call to a String or Iterator variable
+${<prefix>.<variable name>}
+
+// Call to a key in a Dict variable
+${<prefix>.<variable dict name>[key][sub-key]}
+
+----
+
```

Comment:
You can add a note about the `$(...)` alternative but deprecated syntax on Unix agents.

Reply:
it is on line 10 ?